src/xrpld/ledger/View.h

```diff
@@ -76,6 +76,9 @@ enum class SkipEntry : bool { No = false, Yes };
 [[nodiscard]] bool
 hasExpired(ReadView const& view, std::optional<std::uint32_t> const& exp);
 
+[[nodiscard]] bool
+noDefaultRipple(ReadView const& view, Issue const& issue);
```

Comment:
IMO better don't use inversion in function name and meaning, it is harder to follow. How about `isDefaultRipple()` and using` !isDefaultRipple()` ?